images/scripts/rhel4edge.bootstrap

```diff
@@ -55,8 +55,33 @@ if [ "$STATUS" != "SUCCESS" ]; then
     echo "Failed to build rhel4edge-$timestamp"
     exit 1
 fi
+ISO_FILE="rhel4edge-$timestamp.iso"
+curl --netrc-file /secrets/crc_passwd -L "https://console.redhat.com/api/edge/v1/storage/isos/$ID" -o "$ISO_FILE"
+
+# ==========================================
+# customize fleet.ks file inside ISO, to be able to do installation without user interaction
+DIR_NAME=$(mktemp -d -p $BASE)
+FLEET_FILE="$DIR_NAME/fleet.ks"
+
+mv "$ISO_FILE" "$DIR_NAME/fleet_source.iso"
+chmod 775 "$DIR_NAME"
+# extract fleet.ks to directory
+xorriso -osirrox on -indev "$DIR_NAME/fleet_source.iso" -extract /fleet.ks "$DIR_NAME/fleet.ks"
```

Comment:
You can also do this with `isoinfo -x`, that tool is already in the tasks container.